synapse/tests/test_lib_stormlib_iters.py

```diff
@@ -0,0 +1,68 @@
+import synapse.exc as s_exc
+
+import synapse.tests.utils as s_test
+
+class StormLibItersTest(s_test.SynTest):
+
+    async def test_stormlib_iters_zip(self):
```

Comment:
Might be good to add a test for one of the genrs raising an exception.  The ExceptionGroup repr is not great, so I'm wondering if we could do something to make that a bit more clear.